NEXT_CHANGELOG.md

```diff
@@ -27,6 +27,64 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 
 ## ❗ BREAKING ❗
 
+### Configuration: Update metrics and healthcheck web endpoints, and make them configurable ([#1500](https://github.com/apollographql/router/issues/1500))
+
+The web endpoints exposed by the router listen to 120.0.0.1 by default, and the ports and paths for health check and prometheus have changed.
```

Comment:
```suggestion
The web endpoints exposed by the router listen to 127.0.0.1 by default, and the ports and paths for health check and prometheus have changed.
```

Reply:
fixed in [8d9f60d](https://github.com/apollographql/router/pull/1718/commits/8d9f60d57547bd389b94ea5cc8632392bc9d7659)